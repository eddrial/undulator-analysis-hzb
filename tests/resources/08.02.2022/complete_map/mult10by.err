 error report for the data set mult10by.dvm        
--------------------------------------------
      
         128   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1591 data points 
        out of totally   2557 data points
 the rms value of the residuals is  0.45407E-02
 the maximum residual is  0.16482E-01
 the rms value of the lambda/2-filtered data is  0.35963E-01
 the maximum value of the lambda/2-filtered data is  0.27681E+00
