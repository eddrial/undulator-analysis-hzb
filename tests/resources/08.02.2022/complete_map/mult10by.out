mult10by.dvm
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1  1.0135493E+00
   2 -1.0374896E-02
   3 -9.7340494E-03
   4 -1.6751251E-04
   5 -2.4222826E-04
   6  5.5981567E+01
   7  1.0438889E+03
   8  1.0436370E+03
   9  1.0439744E+03
  10  8.0579810E-02
  11 -4.0658045E+00
 number of iterations:   11
     sumsq:  3.28039E-02
       rms:  4.54075E-03  dimension of magnetic field
       rms:  4.48001E-01  percent of b_eff
   rms_ext:  3.74791E-01  percent of b_eff
 B_eff =   1.0136E+00
 B_max =   9.9389E-01
 K_eff =    5.301
 lamda_1 (nm) =     38.23616

        eigenvectors in parameter space

 error                         direction
     1.23,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
     0.66,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
     0.05,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
     0.03,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     0.00,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.00,  0.70 -0.45 -0.49 -0.20 -0.16 -0.07  0.00  0.00  0.00  0.00  0.00
     0.00,  0.12  0.06 -0.37  0.79  0.47  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.67  0.65  0.35  0.02 -0.01  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.15 -0.34  0.40 -0.28  0.79 -0.03  0.00  0.00  0.00  0.00  0.00
     0.00,  0.15 -0.49  0.59  0.51 -0.36 -0.04  0.00  0.00  0.00  0.00  0.00
     0.00,  0.06 -0.06  0.00 -0.01  0.00  1.00  0.05  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:  -0.01775 Tmm
 kick_2:   0.01775 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.31334E+02
      
 the truncated data set has only   1591 data points 
      
 the rms value of the residuals is  0.45407E-02
 the maximum residual is  0.16482E-01
      
 the rms value of the lambda/2-filtered data is  0.35963E-01
 the maximum value of the lambda/2-filtered data is  0.27681E+00
      
 the rms phase error (all data) is    3.768 deg.
 the maximum phase error (all data) is    9.139 deg.
      
 the rms phase error (extrema) is    4.766 deg.
 the maximum phase error (extrema) is    9.776 deg.
      
 the Walker-rms phase error (all data) is   19.385 deg.
 the Walker-maximum phase error (all data) is   34.958 deg.
      
 the Walker-rms phase error (extrema) is    3.881 deg.
 the Walker-maximum phase error (extrema) is    7.937 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error             3.768110    
 local phase error             2.246127    
 slowly varying phase error    2.976455    
 oscillatory phase error      0.5424793    
 -----------------------------------------
